doc/missing-vz-api.md

```diff
@@ -1,13 +1,13 @@
-# Missing API
+# Missing APIs
 
-This document contains a non-exhaustive list of API which are not used/supported by `vfkit`, but which would be useful to have.
+This document contains a non-exhaustive list of APIs not currently used/supported by `vfkit`, but could be useful to have.
```

Comment:
"but could be useful to have", or "but which could be useful to have"? The latter makes more sense to me, but I'm not a native English speaker.

Reply:
(and I suspect this doc is out of date by now, I don't think I've updated it after macOS 14 release)

Reply:
```suggestion
This document is a non-exhaustive list of unimplemented APIs, which may be worth adding to `vfkit`.
```

How's this?